mtest/CMakeLists.txt

```diff
@@ -41,26 +41,14 @@ if (MSVC)
       set(_all_h_file "${PROJECT_SOURCE_DIR}/all.h")
 endif (MSVC)
 
-if (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
-      find_package(Qt5QuickCompiler)
-      qtquick_compiler_add_resources(qrc_files ${PROJECT_SOURCE_DIR}/mtest/mtest.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MScore.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Gootville.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Bravura.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MuseJazz.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Free.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-FreeSerif.qrc
+QT5_ADD_RESOURCES(qrc_files ${PROJECT_SOURCE_DIR}/mtest/mtest.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MScore.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Gootville.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Bravura.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MuseJazz.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Free.qrc
+      ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-FreeSerif.qrc
       )
-else (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
-      QT5_ADD_RESOURCES(qrc_files ${PROJECT_SOURCE_DIR}/mtest/mtest.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MScore.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Gootville.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Bravura.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-MuseJazz.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-Free.qrc
-            ${PROJECT_SOURCE_DIR}/mscore/musescorefonts-FreeSerif.qrc
-      )
-endif (Qt5Widgets_VERSION VERSION_GREATER_EQUAL "5.11.0")
```

Comment:
@Jojo-Schmitz : Apologies, I reverted your change here. It was not working for me. The linker was complaining that it cannot find the init functions like (`q_init_resource_mtest`). Unfortunately, I don't have enough knowledge of QT to make a proper fix... 🙁

Reply:
Which Qt version are you using? MuseScore officislly still uses 5.9, but I'm building with 5.12 and have no issues with this

Reply:
I am building with Qt 5.12.9  for MinGW on 64 bits.

Reply:
Me too, and it works

Reply:
The only thing I can think of that might be related is: Have you tried to remove all build files and start a build from scratch? If I understand correctly, the `.qrc` files are missing from the build directory and if you have them from a previous build then it would work. Unfortunately, like I said I don't know much here so it may be that if you try to delete all build files you will just wait an hour to rebuild everything and it will work for you and it is just me doing something wrong. What's more weird to me is that for the build Cmake file I didn't have to revert it, but just for the mtest CMake...